src/refactorings/change-signature/change-signature.test.ts

```diff
@@ -25,6 +25,15 @@ describe("Change Signature", () => {
             return a + b;
           }`
       },
+      {
+        description: "when there are a arrow function",
```

Comment:
Shouldn't this test be moved to the new "testEach" that is about arrow functions?

Reply:
Is a duplicated test. 

Removed at https://github.com/nicoespeon/abracadabra/pull/747/commits/3579f40d0279cb13544161575aabacf263c42ca2

Nice catch!